File: Geometri C++/Parallelogram.cpp
#include "Parallelogram.h"
#include <math.h>

double Parallelogram::GetAngle()
{
	return angle;
}

void Parallelogram::SetAngle(double angle)
{
	this->angle = angle;
}

Parallelogram::Parallelogram(double a, double b, double angle) : Shape(a, b)
{
	this->angle = angle;
}

double Parallelogram::GetCircumference()
{
	return (a * 2) + (b * 2);
}
double Parallelogram::GetArea()
{
	return a * b * sin(angle * 3.1415 / 180);
}

File: Geometri CSharp/Geometri/Parallelogram.cs
using System;

public class Parallelogram : Shape
{
    private double angle;
    public double Angle
    {
        get { return angle; }
        set { angle = value; }
    }

    public Parallelogram(double a, double b, double angle) : base(a, b)
    {
        Angle = angle;
    }

    public override double GetCircumference() 
    {
        return (a * 2) + (b * 2);
    }

    public override double GetArea() 
    {
        return a * b * Math.Sin(angle * 3.1415 / 180);
    }
}
